misk/src/main/kotlin/misk/security/ssl/CertificatesModule.kt

```diff
@@ -0,0 +1,61 @@
+package misk.security.ssl
+
+import com.google.inject.TypeLiteral
+import misk.scope.ActionScoped
+import misk.scope.ActionScopedProvider
+import misk.scope.ActionScopedProviderModule
+import misk.security.x509.X500Name
+import java.security.cert.X509Certificate
+import javax.inject.Inject
+import javax.servlet.http.HttpServletRequest
+
+/** Installs support for accessing client certificates */
+internal class CertificatesModule : ActionScopedProviderModule() {
+  override fun configureProviders() {
+    bindProvider(
+        type = certificateArrayType,
```

Comment:
do you have to specify the parameter names or are you just doing it for readability?